emoji-ui/src/components/emojiItem.css

```diff
@@ -1,18 +1,18 @@
-.Emoji-container {
+.emoji-container {
   display: inline-block;
   border-radius: 10px;
   padding: 15px;
 }
 
-.Emoji-container:hover, .Emoji-container-selected {
+.emoji-container:hover, .Emoji-container--selected {
```

Comment:
this is now inconsistent: `Emoji-container--selected` vs `emoji-container1 **<-- should probably fix this as it also doesn't match the dom now.**

Reply:
updated the pr. thanks for catching it 👍